plugin.audio.somafm/addon.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.somafm"

```

Comment:
Please a correct SPDX License identifer (https://spdx.org/licenses/) in a `<license>` tag.